Code/Mantid/docs/source/algorithms/MonitorEfficiencyCorUser-v1.rst

```diff
@@ -0,0 +1,59 @@
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
```

Comment:
Would be better if this said something about what log values are expected. Ei etc.